src/components/Home.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
+import { Button } from 'react-bootstrap';
 
 const Home = () => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="text-center">
+      <h1>CYF Quiz</h1>
+      <p className="mb-0 py-0 h3 mt-4">Hello Coder !</p>
+      <p className="mb-0 py-0 h3">Are you ready for the next challenge?</p>
+      <p className="mb-0 py-0 h3">Take a quiz!</p>
+      <ul className="list-unstyled my-4">
+        <li><Button className="mb-2 btn-block w-25 mx-auto font-weight-bold" href="quiz">HTML</Button></li>
```

Comment:
Some of these CSS classes don't make sense with each other — for example, `btn-block` takes up the full width of the window which means `mx-auto` does nothing, and `w-25` makes the width 25% of the window which overwrites `btn-block` and makes `btn-block` redundant. Always try to understand what the code does and how it affects other parts of the code so that you don't write more code than necessary. One thing you can do when coding is delete a class to see how it affects the code and you can see if it's necessary to keep in.

I suggest we remove `btn-block`, `w-25`, and `mx-auto` for all the buttons. When the button is 100% of the window, it looks too wide. When the button is 25% of the window, then when the screen is the size of the mobile the button is too small. I don't think the buttons need to be the same width as each other, but it would look better if they were a bit wider. We could add some horizontal padding with `px-4`?

Reply:
Yes, I was trying to fix the width of the buttons, I will delete them.
 I added px-4 as well.

Reply:
If I may ask, is is better using className for styling instead of creating a scss file for each component !

Reply:
In reality, it depends on the project since different projects do CSS differently.

For this project we are CSS utility-first, which means we always try to use a CSS utility (https://getbootstrap.com/docs/4.4/utilities/) when we can for the UI, but when there's no CSS utility that can do the styling for us then we write new CSS in a SCSS file. The reason for being utility-first is so that we can reuse CSS that already exists and avoid adding more and more CSS which increases download size to the user